integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataRecordSetProvider.java

```diff
@@ -142,13 +142,14 @@ private void fillFilter2QueryModel(QueryModel queryModel,
       }
 
       List<Object> singleValues = new ArrayList<>();
-      List<Expression> rangeFilter = new ArrayList<>();
+
+      List<Expression> disjuncts = new ArrayList<>();
+
       for (Range range : domain.getValues().getRanges().getOrderedRanges()) {
-        checkState(!range.isAll()); // Already checked
```

Comment:
@chenliang613 Actually this code is fix for IS NOT NULL  operator results null values. 
[!CARBONDATA-980](https://issues.apache.org/jira/browse/CARBONDATA-980)
For now i will be removing this code from here and will raise different PR for that.